terraform/modules/bosh_vpc/variables.tf

```diff
@@ -41,7 +41,7 @@ variable "nat_gateway_instance_type" {
 }
 
 variable "nat_gateway_ami" {
-  default = "ami-e8ab1489"
+  default = "ami-6ae2660b"
```

Comment:
Given the convo above about it being a multi step process to zero downtime upgrade these instances I don't think we'd want to autobump this.